content/sensu-go/6.8/operations/maintain-sensu/disaster-recovery.md

```diff
@@ -0,0 +1,348 @@
+---
+title: "Restore a Sensu instance for disaster recovery"
+linkTitle: "Restore a Sensu Instance"
+description: "Learn how to export and back up your Sensu resources and restore a Sensu instance for disaster recovery (DR)."
+weight: 27
+version: "6.8"
+product: "Sensu Go"
+platformContent: false 
+menu:
+  sensu-go-6.8:
+    parent: maintain-sensu
+---
+
+This page describes best practices for making Sensu backups and processes for restoring Sensu instances for disaster recovery.
+
+The disaster recovery processes include steps for creating a backup, but you must make backups *before* you need to use them.
+Read [best practices for backups][2] for more information.
+
+{{% notice warning %}}
+**IMPORTANT**: You may need to adjust the example commands provided in these disaster recovery processes according to your Sensu configuration.<br><br>
+The recommended backup steps do not include events because events may be outdated due to the time elapsed between making backups and restoring resources.
+{{% /notice %}}
+
+## Disaster recovery for PostgreSQL
+
+{{% notice note %}}
+**NOTE**: This process uses [sensuctl dump](../../../sensuctl/back-up-recover/) to create backups.
+When you export users with sensuctl dump, passwords are not included &mdash; you must add the [`password_hash`](../../../sensuctl/#generate-a-password-hash) or `password` attribute back to exported user definitions before you can restore users with sensuctl create.
+Also, sensuctl create does not restore API keys from a sensuctl dump backup.<br><br>
+We suggest backing up API keys and users in a separate file that you can use as a reference for granting new API keys and adding the `password_hash` or `password` attribute to user definitions.
+{{% /notice %}}
+
+If you use PostgreSQL for your Sensu instance, follow these steps to create a backup and restore your Sensu configuration:
+
+1. Create a backup folder:
+
+   {{< code shell >}}
+mkdir backup
+{{< /code >}}
+
+2. Create a backup of the entire configuration, except events, PostgreSQL configuration, API keys, and users, for all namespaces:
+
+   {{< code shell >}}
+sensuctl dump all \
+--all-namespaces \
+--omit core/v2.Event,store/v1.PostgresConfig,core/v2.APIKey,core/v2.User \
+--format wrapped-json \
+--file backup/config_backup.json
+{{< /code >}}
+
+3. Export the PostgreSQL configuration:
+
+   {{< code shell >}}
+sensuctl dump store/v1.PostgresConfig \
+--format wrapped-json \
+--file backup/psql_config_backup.json
+{{< /code >}}
+
+4. Export the API keys and users, for all namespaces:
+
+   {{< code shell >}}
+sensuctl dump core/v2.APIKey,core/v2.User \
+--all-namespaces \
+--format wrapped-json \
+--file backup/apikeys_users_backup.json
+{{< /code >}}
+
+5. Reconfigure the PostgreSQL database.
+
+6. Start a new Sensu backend or cluster and confirm that it is running:
+
```

Comment:
I added these steps because my psql test scenario in Skunkworks uses them to start up, but I'm not sure whether they're required in all cases. Can anyone confirm that you must start the Sensu backend/cluster with etcd as usual before cutting over to Postgres?